Fee Market Meetings/Meeting 06.md

```diff
@@ -0,0 +1,309 @@
+# EIP-1559 Implementers' Call 6 Notes <!-- omit in toc -->
+
+### Meeting Date/Time: Thursday, Nov 5th, 14:00 UTC <!-- omit in toc -->
+
+### Meeting Duration: 1 hr <!-- omit in toc -->
+
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/215) <!-- omit in toc -->
+
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=LgvUnCdMXQg) <!-- omit in toc -->
```

Comment:
The video got broken into two parts. Could you also link the second part? https://www.youtube.com/watch?v=oa4r8m-a2yM

Reply:
```suggestion
### Audio/Video of the meeting: [Part 1](https://www.youtube.com/watch?v=LgvUnCdMXQg), [Part 2](https://www.youtube.com/watch?v=oa4r8m-a2yM) <!-- omit in toc -->
```